hydrocarbon_problem/api/aspen_api.py

```diff
@@ -0,0 +1,53 @@
+from typing import Tuple
+
+from hydrocarbon_problem.api.api_base import BaseAspenDistillationAPI
+from hydrocarbon_problem.api.types import StreamSpecification, ColumnInputSpecification, \
+    ColumnOutputSpecification, ProductSpecification
+
+
+class AspenAPI(BaseAspenDistillationAPI):
+    def __init__(self):
+        pass
```

Comment:
Here you will add something like:
`self._simulator = Simulator("aspen_file.bkp")`
which you can then use inside each of the methods by calling `self._simulator`